src/meta/src/manager/cluster.rs

```diff
@@ -226,28 +225,47 @@ where
 
     pub async fn update_schedulability(
         &self,
-        host_address: HostAddress,
+        worker_ids: &[u32],
         is_unschedulable: bool,
-    ) -> MetaResult<WorkerType> {
+    ) -> MetaResult<Vec<WorkerType>> {
```

Comment:
Why returning worker type here? It seems like the return value are never used.

Reply:
Will change it to `MetaResult<()>`